TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/Shooter.java

```diff
@@ -0,0 +1,22 @@
+package org.firstinspires.ftc.teamcode.subsystems;
+
+import com.arcrobotics.ftclib.command.SubsystemBase;
+import com.arcrobotics.ftclib.hardware.motors.Motor;
+import com.arcrobotics.ftclib.hardware.motors.MotorEx;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+public class Shooter extends SubsystemBase {
+    private Motor kuunav;
+
+    public Shooter(HardwareMap hmap, String name){
+        kuunav = new Motor(hmap, name);
+    }
+
+    public void shot(){
+        kuunav.set(1);
+    }
+
+    public void stoop(){
+        kuunav.set(0);
+    }
+}
```

Comment:
Your shooter should not create a new Motor reference because Java handles these poorly. Instead we can utilize what I like to call "pseudo-pointers," since under-the-hood these values function as pointers but don't really act like them post-abstraction.
Disregarding that discussion, it would be better practice if you passed a Motor object into the constructor and created that motor in your OpMode.